website/src/components/CommandBox/index.css

```
@@ -0,0 +1,110 @@
+}
+
+.command-box__command {
+  display: flex;
+  align-items: center;
+  flex: 1;
+  min-width: 0;
+  padding: 1.1rem 1.5rem;
+}
+
+.command-box__code {
+  font-family: 'Monaco', 'Menlo', 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)